charts/workflow/values.yaml

```diff
@@ -122,13 +122,12 @@ database:
     host: "database host"
     port: "database port"
 
-logger:
-  redis:
-    # Configure the following ONLY if using an off-cluster Redis instance for logger
-    db: "0"
-    host: "redis host"
-    port: "redis port"
-    password: "redis password" # "" == no password
+redis:
+  # Configure the following ONLY if using an off-cluster Redis instance for logger
+  db: "0"
+  host: "redis host"
+  port: "redis port"
+  password: "redis password" # "" == no password
```

Comment:
Looks like this is taken into account when logger's redis is set to 'on-cluster'. I get the following error:
```
*** FATAL CONFIG FILE ERROR ***
Reading the configuration file, at line 396
>>> 'requirepass redis password'
Bad directive or wrong number of arguments
```